docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-snippet/icon-snippet.component.less

```diff
@@ -0,0 +1,20 @@
+:host {
+   display: inline-flex;
+    padding: 10px 0;
+    width: 100%;
+
+    uxd-snippet {
+      display:flex;
```

Comment:
Space missing between property name and value

Reply:
space added